internal/provider/resource_tfe_workspace.go

```diff
@@ -98,6 +98,12 @@ func resourceTFEWorkspace() *schema.Resource {
 				Default:  false,
 			},
 
+			"auto_apply_run_trigger": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  false,
```

Comment:
Is the default needed for some behavior you are trying to elicit? I wonder if optional is enough

Reply:
Good question! I was following the existing behavior of `auto_apply`, since it's basically the exact same setting.

Torn between my user brain saying "unmanaged is fine," my dev brain saying "Optional + Computed always causes weirdness on unset, love 2 avoid that when possible," and both brains saying "just be consistent with the other thing it's basically identical to."